hledger/hledger.cabal

```diff
@@ -45,6 +45,7 @@ extra-source-files:
     embeddedfiles/hledger-web.1
     embeddedfiles/hledger-web.txt
     embeddedfiles/hledger-web.info
+    ../shell-completion/hledger-completion.bash
```

Comment:
I am suspicious, I bet .. paths are problematic here.

Reply:
There are completions for hledger-ui and hledger-web as well, I guess ?

Would it make sense to put each tool's completions in the respective package, or is it better to keep them all together ?

Reply:
Yes, cabal check warns about such paths and hackage rejects them.

Reply:
> Yes, cabal check warns about such paths and hackage rejects them.

Thanks for checking.

> There are completions for hledger-ui and hledger-web as well, I guess ?

I just looked into `hledeger-completion.bash`. It provides basic completion for "extensions" hledger-ui and hledger-web. But it's one bash completion script and Make process for all hledger binaries.

> Would it make sense to put each tool's completions in the respective package, or is it better to keep them all together ?

I guess we could put the whole folder `shell-completion` into the hledger package. @alerque, what do you think?

Reply:
FYI I'll do a minor release soon.

Reply:
@schoettl I guess that depends on what "package" you are talking about? A source dist package? If so then yes.

Reply:
@alerque I'm talking about a haskell package, i.e. a folder containing a package.yaml. So I would move the shell-completion folder into the hledger folder to get it packaged in hledger-1.22.tar.gz.

Reply:
Then yes — given that tarball is the source package used by at least some distros to compile their packaging, yes it needs to have the whole kit & kaboodle to make completions so that they make their way to end users.